CommandLine/Accept.cs

```diff
@@ -86,6 +86,29 @@ public static class Accept
                 return null;
             }));
 
+        public static ArgumentsRule LegalFilePathsOnly(
```

Comment:
Yeah, the name reads weird when it gets chained in the command. Could it read WithLegalFilePathsOnly?